_includes/homepage-features-icon-band.html

```diff
@@ -12,16 +12,16 @@ <h3><a href="/container-first">Container First</a></h3>
       <p>Quarkus tailors your application for GraalVM and HotSpot. Amazingly fast boot time, incredibly low RSS memory (not just heap size!) offering near instant scale up and high density memory utilization in container orchestration platforms like Kubernetes. We use a technique we call compile time boot.</p>
     </div>
     <div class="width-4-12 width-12-12-m contrib-block">
-      <img class="light-only" src="{{site.baseurl}}/assets/images/about/icon-reactive.svg">
-      <img class="dark-only" src="{{site.baseurl}}/assets/images/about/icon-reactive-dark.svg">
-      <h3><a href="/continuum">Reactive Core</a></h3>
-      <p>Built on a robust reactive core, Quarkus ensures fast and efficient performance, supporting the development of a wide variety of modern applications.</p>
+      <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-performance.svg">
```

Comment:
I would not have put performance in the second item, on the second line. 

I know it's a core strength, but, that's not going to attract lots of developers (otherwise, many languages would not be as popular as they are...).